bin/runtime/src/lib.rs

```diff
@@ -518,9 +519,13 @@ impl_runtime_apis! {
         }
     }
 
-    impl primitives::AlephApi<Block> for Runtime {
-        fn authorities() -> Vec<AlephId> {
-            Aleph::authorities()
+    impl primitives::AlephApi<Block, AlephId, BlockNumber> for Runtime {
+        fn current_session() -> Option<AuthoritySession<AlephId,BlockNumber>> {
+            Aleph::current_session()
+        }
+
+        fn next_session() -> Option<AuthoritySession<AlephId,BlockNumber>> {
```

Comment:
If possible I would prefer this to be a vector of next sessions even if always only 1 will be know in advance

Reply:
I would even say, the most convenient would be something like
`get_session(SessionId) -> Option<AuthoritySession...>`
but @kostekIV is the one who knows the best :)

Reply:
+1 for querying based on SessionID, looks like the cleanest API to use and write. Although what should be the behaviour if we ask for session `k` in a block in which we are already in session `k + 1 + something`?  Should it return None? If not we should probably store some map from sessionID to AuthoritySession in the pallet.